samples/client/petstore/ruby/Gemfile.lock

```diff
@@ -7,40 +7,43 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    ast (2.4.1)
```

Comment:
thanks for the PR.

why are there changes to Gemfile.lock?

Reply:
Circle CI was failing with the newer Ubuntu image. Probably because it has a newer version of Bundler. Deleting and re-generating the lockfiles helped.

```
Bundler 2.5.3 is running, but your lockfile was generated with 1.17.2. Installing Bundler 1.17.2 and restarting using that version.
Fetching gem metadata from https://rubygems.org/.
Fetching bundler 1.17.2
Installing bundler 1.17.2
/opt/circleci/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/bundler-1.17.2/lib/bundler/shared_helpers.rb:272:in `search_up': undefined method `untaint' for an instance of String (NoMethodError)

      current  = File.expand_path(SharedHelpers.pwd).untaint
                                                    ^^^^^^^^
```

https://app.circleci.com/pipelines/github/OpenAPITools/openapi-generator/29141/workflows/2723157b-b3c5-40ff-a8eb-48567400b615/jobs/93653